### ISSUE TITLE
Server Problem on Demo Site?

### ISSUE THREAD

**Issue by vidhill** [mozilla/serviceworker-cookbook#196]
I'm seeing 503 errors on the demo pages:

For example: https://serviceworke.rs/push-simple_demo.html

https://serviceworke.rs/push-simple/register  503 error
https://serviceworke.rs/push-simple/sendNotification?.... 503 error

**Comment by marco-c** [comment]
I'm not seeing 503 errors.
Does the demo work for you? What browser are you using?

The `register` request is empty, I don't see how it could fail:
```JavaScript
  app.post(route + 'register', function() {
    // A real world application would store the subscription info.
  });
```

**Comment by vidhill** [comment]
No, the notification wasn't working, it was on Chrome Canary on Windows (my work machine).

I'm on Chrome (standard ) on Mac now and it's working fine here

- I do still see the 503 responses but they're obviously not affecting it.
Log; [serviceworke.rs-1452805485737.txt](https://github.com/mozilla/serviceworker-cookbook/files/91010/serviceworke.rs-1452805485737.txt)

![image](https://cloud.githubusercontent.com/assets/3375475/12337638/0ebfb0ae-bb03-11e5-9523-3f63a8616b93.png)

-if you want me to check anything on Canary I can check when I'm at work tomorrow..

**Comment by marco-c** [comment]
Thank you, I think I've found the issue and #197 should fix it.
The errors were spurious though, so they shouldn't affect the notification stuff.

Can you open another issue, after #197 is merged, if you still can't receive notifications on Windows?

**Comment by vidhill** [comment]
Yes, 
No problem, 

I'll check it, when it's merged..

**Comment by vidhill** [comment]
Issues still exists, don't expect to be resolved until pull #197 is merged
